pkg/joinserver/joinserver.go

```diff
@@ -53,6 +53,8 @@ type Config struct {
 	JoinEUIPrefixes []*types.EUI64Prefix `name:"join-eui-prefix" description:"JoinEUI prefixes handled by this JS"`
 }
 
+var _ interface{} = jsServer{}
```

Comment:
Remove

Reply:
If what you want to do, is assert that `jsServer` implements an interface `I`, you have to do:
```go
	var _ I = jsServer{}
```
And this belongs in test code